lib/http-bridge/test/integration/Cardano/WalletSpec.hs

```diff
@@ -62,6 +64,22 @@ spec = do
                 unsafeRunExceptT (readWallet wallet wid)
             unless (tip > (SlotId 0 0)) $
                 expectationFailure ("The wallet tip is still " ++ show tip)
+
+        it "The time of the tip is recent" $ \(_, wallet) -> do
```

Comment:
I'm not sure whether "The time of the tip is recent" is true in general. The network can miss blocks for example, or the node could be temporarily disconnected from its peers.